Build/PackageFiles/ExtensionBundleTemplates-2.x.nuspec

```diff
@@ -108,6 +108,9 @@
     <file src="templates/DurableFunctionsActivity-JavaScript/function.json" target="Templates/DurableFunctionsActivity-JavaScript/function.json" />
     <file src="templates/DurableFunctionsActivity-JavaScript/index.js" target="templates/DurableFunctionsActivity-JavaScript/index.js" />
     <file src="templates/DurableFunctionsActivity-JavaScript/metadata.json" target="Templates/DurableFunctionsActivity-JavaScript/metadata.json" />
+    <file src="templates/DurableFunctionsActivity-Python-2.x/function.json" target="Templates/DurableFunctionsActivity-Python-2.x/function.json" />

```

Comment:
same as above target should be `Templates/DurableFunctionsActivity-Python/function.json`

Reply:
Fixed this as well. I see the Durable C# ones have `-2.x`. I'll remove them too.